build.sbt

```diff
@@ -2,19 +2,22 @@ name := "play-slick-advanced"
 
 version := "2.0.0"
 
-scalaVersion := "2.11.7"
+scalaVersion := "2.11.8"
 
-crossScalaVersions := Seq("2.10.6", scalaVersion.value)
+resolvers += "Local Maven Repository" at "file://"+Path.userHome.absolutePath+"/.m2/repository"
```

Comment:
what is the meaning of this? I mean - isn't that basically the default behavior of `resolvers += Resolver.mavenLocal` ? (I think it is)

http://www.scala-sbt.org/0.13/docs/Library-Dependencies.html